impeller/entity/entity_pass.cc

```diff
@@ -338,8 +352,8 @@ bool EntityPass::Render(ContentContext& renderer,
   // and then blit the results onto the onscreen texture. If using this branch,
   // there's no need to set up a stencil attachment on the root render target.
   if (reads_from_onscreen_backdrop) {
-    auto offscreen_target = CreateRenderTarget(
-        renderer, root_render_target.GetRenderTargetSize(),
+    EntityPassTarget offscreen_target = CreateRenderTarget(
+        renderer, root_render_target.GetRenderTargetSize(), required_mip_count,
```

Comment:
I thought we were going to protect this behind a flag for the new blur to keep the benchmark comparison sane?

Reply:
It is behind the new blur flag.  Notice how the tests make sure the mip count is 1.  When you turn the flag off those bump up to 4.

Reply:
So it's doing all the new calculations have test coverage, but the effect on performance should be negligible.

Reply:
Ah, I missed the `FilterContents::kBlurFilterRequiredMipCount`. LGTM